### ISSUE TITLE
Broker Heart Cell Atlas project to ArrayExpress

### ISSUE THREAD

**Issue by mshadbolt** [ebi-ait/hca-ebi-wrangler-central#144]
**Description of the task:**

Carlos has requested raw fastqs be made available to the public, currently the only way of doing this is through ArrayExpress so I will be trying out the process of using the HCA-SCEA tools to convert the metadata spreadsheet to mage-tab and working with AE curators to make a submission.

**Acceptance criteria for the task:**

- [ ] spreadsheet converted to mage-tab and curated
- [ ] files transferred to AE ftp storage
- [ ] AE accession communicated to contributors

**Comment by ESapenaVentura** [comment]
With the changed processes this ticket is a duplicate of this https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/3

**Comment by mshadbolt** [comment]
this will be tracked on the dataset ticket directly

**Comment by mshadbolt** [comment]
This is waiting on fixing the ENA submission

**Comment by mshadbolt** [comment]
We changed the status for this because if the data is made available via DCP soon, we won't need to broker to AE but we will still want to create MAGE-TAB files to be able to broker to the SCEA

**Comment by lauraclarke** [comment]
@mshadbolt What tasks are you tracking with this ticket that aren't part of the standard wrangling process?

**Comment by mshadbolt** [comment]
if you find tickets like these that you don't want on the operations board, feel free to remove the label and I will track on my own board

**Comment by lauraclarke** [comment]
It isn't that this shouldn't be on the operations board or any other board. I am trying to understand what is the difference between this ticket and the one Enrique tagged https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/3 which is tagged as submit to Array Express

Happy for us to be tracking active work in tickets, trying to minimize confusion from apparent ticket duplication.
